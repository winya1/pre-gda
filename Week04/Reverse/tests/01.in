2
hello abcd zyx
aaaaa bbbb ccc
