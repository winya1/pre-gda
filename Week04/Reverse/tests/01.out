Case #1: olleh dcba xyz
Case #2: aaaaa bbbb ccc
